### PR TITLE
Update sys-dm-fts-parser-transact-sql.md

### PR DESCRIPTION
**Pull request by DanielAdeniji**
Reviewed permission requirements.  Noted bare minimum permission needed is "create fulltext catalog".

### CODE REVIEW

**Comment by Court72** [comment]
@rwestMSFT 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by DanielAdeniji** [comment]
Courtney:-

I made an earlier comment on this subject.

It was rejected.

Can you please help me shepherd this request.

My hope is that hopefully it will not be summarily rejected in quick haste.

If there are others within the Microsoft Organization that I should reach out to, please help advise.

And, I will do so.

Quick rejection discourages community participation.

And, that runs counter to the good spirit of this beautiful platform.

BTW, no other organization comes to the platform Microsoft has.

Daniel
________________________________

File: docs/relational-databases/system-dynamic-management-views/sys-dm-fts-parser-transact-sql.md [hunk 1]
```diff
@@ -118,7 +118,9 @@ sys.dm_fts_parser('query_string', lcid, stoplist_id, accent_sensitivity)
   
 ## Permissions  
  Requires membership in the **sysadmin** fixed server role and access rights to the specified stoplist.
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-fts-parser-transact-sql.md:120]: Please make the requisite change to this line.

For example:

```suggestion
Requires CREATE FULLTEXT CATALOG permission, and access rights to the specified stoplist.
```

File: docs/relational-databases/system-dynamic-management-views/sys-dm-fts-parser-transact-sql.md [hunk 2]
```diff
@@ -118,7 +118,9 @@ sys.dm_fts_parser('query_string', lcid, stoplist_id, accent_sensitivity)
   
 ## Permissions  
  Requires membership in the **sysadmin** fixed server role and access rights to the specified stoplist.  
-  
+ 
+ Verified that the "create fulltext catalog" permission is the bare minimum permission needed.
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-fts-parser-transact-sql.md:122]:
```suggestion
```

File: docs/relational-databases/system-dynamic-management-views/sys-dm-fts-parser-transact-sql.md [hunk 3]
```diff
@@ -118,7 +118,9 @@ sys.dm_fts_parser('query_string', lcid, stoplist_id, accent_sensitivity)
   
 ## Permissions  
  Requires membership in the **sysadmin** fixed server role and access rights to the specified stoplist.  
-  
+
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-fts-parser-transact-sql.md:121]:
```suggestion
```

**Review by rwestMSFT — CHANGES_REQUESTED** [review]
@DanielAdeniji We really do appreciate your contributions. My notes in this PR are a function of how we process pull requests, so hopefully you will receive them as they are intended: to make your contributions even more valuable.

Thank you.

**Comment by DanielAdeniji** [comment]
Randolph:-

Yes, we are in the same space.

Thanks for making every effort.

Daniel
________________________________

**Comment by rwestMSFT** [comment]
Daniel, owing to the merge conflicts, I will incorporate your change (as discussed on the other PR you raised) into an internal PR. When that one is merged, which will be shortly, this one will automatically close, and your changes will be live by this evening Pacific time.

**Pull request closed without merging**